### PR TITLE
Update LiveSync docs to include MongoDB connector

### PR DESCRIPTION
**Pull request by zknill**
Update the LiveSync docs to include the MongoDB connector. Now that LiveSync contains two flavours -- MongoDB, and Postgres connector with Models SDK -- the page structure has been split to make this difference clear to readers. Notably; that if you want to use the Models SDK then you need to use it with MongoDB.

Add the docs for MongoDB.

Also:
- coalesce some of the page structure for Postgres & Models.
- renames so that we aren't referring to just one 'Database connector' and instead reference the underlying database that the connector is connected to.
- add redirects for the old page structure to the new pages

### CODE REVIEW

File: content/livesync/mongodb-connector.textile [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+The LiveSync MongoDB Database connector built on top of Ably Pub/Sub channels to distribute document from a MongoDB collection to end users at scale.
+
+The MongoDB integration can be used to distribute document state to large numbers of subscribing clients in realtime as changes occur,
+with features like "connection-recovery":https://ably.com/docs/connect/states, "exactly-once delivery":https://ably.com/blog/achieving-exactly-once-message-processing-with-ably and "ordering guarantees":https://faqs.ably.com/reliable-message-ordering-for-connected-clients out of the box.
+
+The MongoDB ChangeStream integration can be enabled through an Ably integration rule
+
+h2(#how-it-works). How it works
+
+LiveSync MongoDB integration rule consumes document changes from the customer’s MongoDB Atlas deployment via the MongoDB "ChangeStreams API":https://www.mongodb.com/docs/manual/changeStreams/.
```
Comment (paddybyers) [inline content/livesync/mongodb-connector.textile:16]: Just Atlas?

File: content/livesync/mongodb-connector.textile [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+The integration rule exists as a “database connector” component that is entirely provisioned and managed by Ably.  The database connector is fault tolerant by default and integration rules automatically fail-over to a new worker node in the case of failures. Additionally, the database connector can scale out horizontally with a preconfigured provisioned capacity.
```
Comment (paddybyers) [inline content/livesync/mongodb-connector.textile:17]: Why 2 spaces after `.` ?

File: content/livesync/mongodb-connector.textile [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+Whenever a change event is received over the ChangeStreams API it is published to an Ably channel.  When configuring the rule, you can specify how change events are mapped to individual Ably channels.  Clients can then subscribe to database changes by subscribing to Ably channels.  Ably "Auth":https://ably.com/docs/auth and "Capabilities":https://ably.com/docs/auth/capabilities control which channels a client can interact with.
+
+
+h2(#integration-rule). Integration rule
+
+h3(#create). Creating the rule
+
+There are two ways to create a Kafka rule:
```
Comment (paddybyers) [inline content/livesync/mongodb-connector.textile:26]: Kafka?

File: livesync-channel-architecture.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+changes in the UI while the confirmation goes to a roundtrip through the
+database. When confirmed, the original user's UI update can be confirmed
+and all other clients can also see that update without needing to
+refresh the page.
+
+### Known limitations
+
+LiveSync's current design might be limiting to data models based on
+graph data structures. As each data model is meant to work with a
+channel of it's own, any application changes that apply to multiple data
```
Comment (paddybyers) [inline livesync-channel-architecture.md:25]: `it's` -> `its`

File: livesync-channel-architecture.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+## Use cases
+
+LiveSync can benefit a wide range of applications including Customer
```
Comment (paddybyers) [inline livesync-channel-architecture.md:3]: Why is this hard-wrapped?

**Comment by zknill** [comment]
TODO: MongoDB priviledges

**Comment by zknill** [comment]
Closing, as this is not ready and I don't know how to down-grade to a draft

**Pull request closed without merging**